code/__HELPERS/_macros.dm

```diff
@@ -5,8 +5,30 @@
 // fun if you want to typecast humans/monkeys/etc without writing long path-filled lines.
 #define ishuman(A) istype(A, /mob/living/carbon/human)
 
+#define isjusthuman(A) (ishuman(A) && A.species && istype(A.species, /datum/species/human))
```

Comment:
An istype is also a null check because null can't have a type so no, no issue here.